inference-engine/tests/functional/shared_test_classes/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ addIeTarget(
                 pugixml::static
                 funcTestUtils
                 ngraphFunctions
+                inference_engine_legacy
```

Comment:
why do we need it? can we remove legacy usage from shared tests?

Reply:
We need it, because conv_eltwise_fusion.cpp uses pass::ConvertConvolutions and pass::ConvFusion. If we move these passes to legacy - we have two ways:  add legacy dependence in shared_test_classes/CMakeLists.txt or remove these passes usage from conv_eltwise_fusion.cpp. But when I tried to remove them - I saw red CI, because arm-plugin also depends on them. So, Gleb and I decided to add legacy dependence there.

Reply:
We need it, because conv_eltwise_fusion.cpp uses pass::ConvertConvolutions and pass::ConvFusion. If we move these passes to legacy - we have two ways: add legacy dependence to shared_test_classes/CMakeLists.txt or remove usage of these passes from conv_eltwise_fusion.cpp. If we remove them - we need to change tests suites in subgraph_tests/conv_eltwise_fusion.cpp. So, when I tried it - I saw red CI because arm plugin also uses these tests. Because of it, Gleb and I decided to add legacy dependence there.